lib/matplotlib/ticker.py

```diff
@@ -907,7 +909,6 @@ def __init__(self, base=10.0, labelOnlyBase=False,
                 minor_thresholds = (1, 0.4)
         self.minor_thresholds = minor_thresholds
         self._sublabels = None
-        self._linthresh = linthresh
```

Comment:
that breaks backcompat if someone was passing in `linthresh` -- if you want to maintain compat, I think you need to keep the logic for the symlog case in set_locs for the time being.